starter/src/main/java/io/javaoperatorsdk/operator/springboot/starter/OperatorAutoConfiguration.java

```diff
@@ -61,25 +60,23 @@ public boolean checkCRDAndValidateLocalModel() {
   @ConditionalOnMissingBean(Operator.class)
   public Operator operator(
       KubernetesClient kubernetesClient,
-      List<ResourceController<?>> resourceControllers,
-      Optional<ObjectMapper> objectMapper) {
```

Comment:
Why did you remove the option to configure the ObjectMapper?

Reply:
The ObjectMapper constructor option was removed with 1.9.0. I may have overlooked potential other new constructor options.

Reply:
Ah, yes, sorry! The `ObjectMapper` is now configured via the `ConfigurationService`.